From 5082c141cf5d83ff6333c543af6fc3916101b7b6 Mon Sep 17 00:00:00 2001
From: Evelina Sarkisyan <evelina.sarkisyan.1990@mail.ru>
Date: Sat, 10 Oct 2020 20:10:37 +0300
Subject: [PATCH] updated firstFile 1

---
 firstFile.txt | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/firstFile.txt b/firstFile.txt
index e69de29..4682fd0 100644
--- a/firstFile.txt
+++ b/firstFile.txt
@@ -0,0 +1,10 @@
+Line 1
+Line 2
+Line 3
+Line 4
+Line 5 updated
+Line 6 updated
+Line 7
+Line 8
+Line 9
+Line 10
-- 
2.28.0.windows.1

